data/entities.yaml

```diff
@@ -1289,7 +1289,7 @@
     `watch : dir -> cmd unit` will mark an adjacent (in the specified direction) location of interest to monitor for placement or removal of items.
     A subsequent call to `wait` will be interrupted upon a change to the location.
   properties: [portable]
-  capabilities: [wakeself]
+  capabilities: [timerel, wakeself]
```

Comment:
If we're going to add the `timerel` capability to the `watch` device (which is fine) then we should also mention it in the description (i.e. "Enables robots to use the `watch` and `wait` commands").